wdl/FilterBatchSamples.wdl

```diff
@@ -9,8 +9,12 @@ import "IdentifyOutlierSamples.wdl" as identify_outliers
 workflow FilterBatchSamples {
   input {
     String batch
-    Array[File?] vcfs  # in order of algorithms array: ["manta", "delly", "wham", "melt", "depth"]. To skip one, use null keyword - ex. ["manta.vcf.gz", null, "wham.vcf.gz", null, "depth.vcf.gz"]
-    Array[File?] sv_counts  # one SV counts file from PlotSVCountsPerSample per VCF in the same order
+    File? sites_filtered_manta_vcf
+    File? sites_filtered_delly_vcf
+    File? sites_filtered_wham_vcf
+    File? sites_filtered_melt_vcf
+    File? sites_filtered_depth_vcf
+    Array[File?]? sv_counts_array  # one SV counts file from PlotSVCountsPerSample per VCF in the order ["manta", "delly", "wham", "melt", "depth"]. Enter null for missing SV counts
```

Comment:
I think it would be clearer to just have:
```
    File? manta_counts
    File? delly_counts
    File? wham_counts
    File? melt_counts
    File? depth_counts
```